### ISSUE TITLE
Global refresh for page?

### ISSUE THREAD

**Issue by dtaboada** [ironmansoftware/universal-dashboard#368]
Hi all, was wondering if its even possible for instance to use the New-UDSwitch to control the behavior of  a entire page or individual component refreshes, 

1 use case for my scenario is: I have a bunch of components on a page(charts, grids etc..) with buttons that get all sorts of data, but haven't figured a way of giving the user the option to control the refresh of the data, was thinking New-UDSwitch would be perfect for this but wasn't sure if it was possible before going down that rabbit hole, figured I ask. - Thanks.

**Comment by adamdriscoll** [comment]
This isn't really possible at this point but I've heard this suggestion before. The example of filtering has also been given. Right now the best way to do this is use the $Cache scope and then have the controls refresh on a low interval but it's not ideal.

**Comment by dtaboada** [comment]
Yeah I was more inclined to giving the user control and not have to worry about settings on particular components being displayed. I can see this working very similar to how UDEndpoint schedule hooks up to a UDEndpoint.

**Comment by mattmcnabb** [comment]
I'm wondering if this might be a way to create a theme switcher as well? So we could provide one or more color themes and then use an input of some kind which can refresh the entire dashboard with the new colors? Would be a great use case for me.
